tensorflow/core/kernels/scatter_nd_op.cc

```diff
@@ -25,13 +25,14 @@ limitations under the License.
 #include "tensorflow/core/framework/bounds_check.h"
 #include "tensorflow/core/framework/op_kernel.h"
 #include "tensorflow/core/framework/register_types.h"
+#include "tensorflow/core/framework/resource_mgr.h"
+#include "tensorflow/core/framework/resource_var.h"
```

Comment:
`/tensorflow/core/kernels/scatter_nd_op.cc:29:10: error: module //
/tensorflow/core/kernels:scatter_nd_op does not depend on a module exporting '/tensorflow/core/framework/resource_var.h'

/tensorflow/core/kernels/scatter_nd_op.cc:33:10: error: module //
/tensorflow/core/kernels:scatter_nd_op does not depend on a module exporting '/tensorflow/core/kernels/dense_update_functor.h'`